portafly/src/i18n/locales/en/audience/accounts/listing.yml

```diff
@@ -0,0 +1,108 @@
+
+# Top area
+
+create_account_button: Create account
+create_account_button_description: Button link to create an account
+export_accounts_button: Export all accounts
+export_accounts_button_description: Button link to export all the accounts.
+
+# Main area
+
+## Toolbar
+actions_empty_label: Select accounts first to enable actions
+actions_empty_label_description: Admonition that displays inside the actions dropdown when no items are selected
+actions_dropdownlist: Actions
+actions_dropdownlist_description: String to list all the available actions.
+actions_dropdownlist_options:
+  send_email: Send email
+  change_state: Change state
+
+accounts_filter: Group/organization
+accounts_filter_description: Dropdown list to filter accounts by specified option
+actions_filter_options:
+ by_group: Group/organization
+ by_admin: Admin
+ by_state: State
+ by_state_options:
+  approved: Approved
+  pending: Pending
+  rejected: Rejected
+  suspended: Suspended
+accounts_filter_typeahead: "Filter by {{filter_option}}"
+accounts_filter_typeahead_description: Filter by typeahead a text input. This applies for filters by group/organization and admin
+
+## Header
+accounts_table:
+ aria_label: Accounts table
+ aria_label_description: Aria label of the table of accounts
+ empty_state: Currently, there are no accounts to show. Add accounts.
+ empty_state_description: Message to display when there are no accounts to show, and prompt the user to add accounts
+ group_header: Group/Organization
+ group_header_description: Column listing groups and organizations
+ admin_header: Admin
+ admin_header_description: Column listing administrators
+ signup_header: Signup date
+ signup_header_description: Column listing signup dates
+ applications_header: Applications
+ applications_header_description: Column listing applications linked to accounts, on a 1:1 relationship
+ state_header: State
+ state_header_description: Column listing the state of the accounts
+ #-------# actions_header_plural: available in shared.yml
+ actions_column_options:
+  approve: Approve
+  approve_description: Button link to approve an account
+  activate: Activate
+  activate_description: Button link to perform the activation of an account
+  act_as: Act as
+  act_as_description: Button link enabled for administrators to impersonate an account
+
+# Modal: Send email
+send_email:
+  page: Send email to admins of selected accounts
+  page_description: Body title of contextual window for sending email to administrators of the selected accounts
+  to_label: To
+  to_label_description: Header for email recipients
+  subject_label: Subject
+  subject_label_description: Header for email subject
+  message_label: Message
+  message_label_description: Header to email message
+  #-------# Send button: available in shared.yml
+  #-------# Cancel button: available in shared.yml
+  toasts:
+   send_email_start: Sending email
```

Comment:
@porueesq technically we don't need this message, the modal dialog will auto-dismiss when the request is successful, so the messages have been sent already, and toast message displayed will be `send_email_success`
@josemigallas can you please confirm?

Reply:
See https://github.com/3scale/porta/pull/1928#discussion_r431729026